Automattic/newspack-theme
# feat: add Bluesky to the Social Links menu

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds the [Bluesky](https://bsky.app/) icon to the Social Links menu.

See 1200550061930446-as-1208769260234967

### How to test the changes in this Pull Request:

1. Apply this PR.
2. Add a link to a Bluesky account to a menu assigned to the Social Links location (or a link including the base URL https://bsky.app/). 
3. Add some more icons to the menu if there aren't any already, to compare the sizes.
4. View in the header and footer -- confirm that the Bluesky icon appears (a butterfly) and that it appears scaled well with the other icons:

![CleanShot 2024-11-19 at 15 10 01](https://github.com/user-attachments/assets/abdffb10-e89e-4600-b614-dafe8a192910)

5. Enable the header solid background, and switch the header to a darker colour, confirm that the butterfly switches to white with the other icons (if you already had a solid background, you need to disable it). 

![CleanShot 2024-11-19 at 15 09 32](https://github.com/user-attachments/assets/15f716de-941d-4cb3-acab-e3ce3728a267)

### Other information:
